delta/plugins/search/src/test/scala/ch/epfl/bluebrain/nexus/delta/plugins/search/SearchSpec.scala

```diff
@@ -145,7 +149,14 @@ class SearchSpec
   private val prefix = "prefix"
 
   "Search" should {
-    lazy val search = Search(listViews, aclCheck, esClient, prefix)
+    lazy val search = Search(listViews, aclCheck, esClient, prefix, allSuites)
+
+    val payload         = jobj"""{"size": 100}"""
+    val queryParameters = Query.Empty
```

Comment:
If you called this "get everything" then the intention is much clearer when reading the test